collector-http/src/main/java/zipkin2/collector/otel/http/OtelResourceMapper.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright The OpenZipkin Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package zipkin2.collector.otel.http;
+
+import java.util.function.BiConsumer;
+
+import io.opentelemetry.proto.resource.v1.Resource;
+
+/**
+ * The interface to map OpenTelemetry Resource to Zipkin Span
+ */
+public interface OtelResourceMapper extends BiConsumer<Resource, zipkin2.Span.Builder> {
```

Comment:
ok as collector is java 8, but also begs the question why don't we accept `BiConsumer<Resource, zipkin2.Span.Builder>` instead of this interface?

Reply:
I thought it would be more explicit and easier to use if there was a specific interface when retrieving/registering a bean by type from/to the application context (
For example, https://github.com/openzipkin-contrib/zipkin-otel/pull/19/commits/686a72de98116a04bce690086e455ec0e4ed7404#diff-4eae431b1454fe7f8d134937a0dbb36e6b9eff63844be9f55bbc346a22377a0aR61
or https://github.com/openzipkin-contrib/zipkin-otel/pull/19/commits/686a72de98116a04bce690086e455ec0e4ed7404#diff-79932ed993c34a9076e0e401cebea87ae9f0b9c45c18ebf46eb103739d9fbb03R27 )
, but I don't have any particular preferences.

Reply:
works for me, was just curious